docs/API/content-type/creating-co.md

```diff
@@ -3,106 +3,392 @@ description: How to add Content Objects in Flotiq
+        CURLOPT_HTTPHEADER => [
+                "X-AUTH-TOKEN: YOUR_API_KEY",
+                "content-type: application/json"
+            ],
+        ]);
+        
+        $response = curl_exec($curl);
+        $err = curl_error($curl);
+        
+        curl_close($curl);
+        
+        if ($err) {
+            echo "cURL Error #:" . $err;
+        } else {
+            echo $response;
+        }
+        ```
+
+
+!!! Responses
+
+    === "200 OK"
+
+        Returned when data has been correct, and the object was saved
+
+        ```
+        {
+            "id": "blogposts-456712",
+            "internal": {
+                "contentType": "blogposts",
+                "createdAt": "2021-04-09T13:30:48+00:00",
+                "updatedAt": "2021-04-09T13:30:48+00:00",
+                "deletedAt": ""
+            },
+            "title": "New object",
+            "postContent": "This will be the new <b>content</b>"
+        }
+        ```
+
+    === "400 Validation error"
+
+        Returned when data has not been correct, and the object was not saved
+
+        ```
+        {
+            "id": [
+                "This value is already used",
+                "Must be at least 1 characters long",
+            ],
+            "title": [
+                "The property title is required",
+                "Must be at least 1 characters long",
+                "The property title is required"
+            ],
+            "postContent": [
+                "The property postContent is required",
+                "Must be at least 1 characters long",
+                "The property postContent is required"
+            ]
+        }
+        ```
+
+    === "401 Unauthorized"
+
+        Returned when API key was missing or incorrect
+  
+        ```
+        {
+            "code": 401,
+            "massage": "Unauthorized"
+        }
+        ```
+
+    === "404 Not found"
+
+        Returned when content type definition wasn't found
+
+        ```
+        {
+            "code": 404,
+            "massage": "Not found"
+        }
+        ```
+
+#### Possible validation errors
+
+| Error                                        | Description                                                                                               |
+| -------------------------------------------- | --------------------------------------------------------------------------------------------------------- |
+| This value is already used                   | Send when the property has to be unique, and the value is already used in an existing object of that type |
+| Must be at least 1 characters long           | Send when the property is required, and an empty string was sent in object                                |
+| The property {name} is required              | Send when the property is required and was missing in the sent object                                     |
```

Comment:
Unfortunatelly this is rendered as 
![image](https://user-images.githubusercontent.com/645716/116099406-380e9e80-a6ac-11eb-8a22-5df35e336b30.png)

Reply:
```suggestion
| The property \{name\} is required              | Send when the property is required and was missing in the sent object                                     |
```